packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx

```diff
@@ -89,6 +95,13 @@ export const RepositoriesPage = () => {
         showBetaBadge
         nestedPageType={pageType}
       />
+      {!isNotificationDismissed ? (
+        <Notification pageType={pageType}>
+          {itemsToShow().length === 0
+            ? 'CodeSandbox Projects is now Repositories: an improved git workflow powered by the cloud. '
+            : 'Your CodeSandbox Projects repositories now live here. Repository sandboxes are now listed under Synced sandboxes. You can find your contribution branches on My contributions inside your personal team.'}
```

Comment:
should we quote `My contributions` and `Synced sandboxes` for clarity?

Reply:
I forgot to highlight these pieces of text.